frontend/src/metabase-lib/lib/Question.ts

```diff
@@ -1354,10 +1354,9 @@ class QuestionInner {
   }
 }
 
-export default class Question extends memoizeClass<QuestionInner>(
-  "query",
```

Comment:
We can't memoize `query` until we get rid of the `_update` fn we attach to question instances during the QB render cycle.